starter_code/seed.js

```diff
@@ -1,22 +1,124 @@
 
 (async () => {    
-    
+  // Clear Customers
+  try {
+    await Customer.deleteMany( {} );
+    console.log("All customers have been deleted");
+  } catch (error) {
+    console.log( error );
+  };
+  
+  // Clear Orders
+  try {
+    await Order.deleteMany( {} );
+    console.log("All orders have been deleted");
+  } catch (error) {
+    console.log( error );
+  };
+
+  // Clear Pizzas
+  try {
+    await Pizza.deleteMany( {} );
+    console.log("All pizzas have been deleted");
+  } catch (error) {
+    console.log( error );
+  };
+
+
+  // Create some customers with Faker API
+  const customerPromises = Array(2)
+  .fill(null)
+  .map(() => {
+    const customerData = {
+      firstName: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+      address: {
+        street: faker.address.streetName() + " " + faker.address.streetSuffix(),
+        zipCode: faker.address.zipCode(),
+        city: faker.address.city()
+      }
+    }
+    console.log(`Customer ${customerData.firstName} ${customerData.lastName} added to the database`);
+
+    const customer = new Customer( customerData );
+    return customer.save();
+  });
+
+  let customers = [];
+
   try {
     // seed some customers...
+    customers = await Promise.all( customerPromises );
+    console.log("----------------------------------------------");
+    console.log(`All customers have been stored to the DB`);
+    console.log("----------------------------------------------");
+  }
+  // handle errors in seeding
+  catch(err) { 
+      console.log("[ERROR] Seeding failed: ", err)
+  };
+  // --------------------------------------------------------------
 
-    // e.g. use insertMany to seed in an array of objects...
-    // example: await Customer.insertMany([ obj1, obj2 ]
-    // await insertMany will return an array of all inserted items
-      
+  const pizzaPromises = Array(3)
+  .fill(null)
+  .map(() => {
+    const pizzaData = {
+      name: "Pizza " + faker.random.word(1),
+      price: faker.commerce.price(6, 10, 2)
+    };
+    console.log(`${pizzaData.name} added to the database`);
+
+    const pizza = new Pizza( pizzaData );
+    return pizza.save();
+  });
+
+  let pizzas = [];
+
+  try {
+    // seed some customers...
+    pizzas = await Promise.all( pizzaPromises );
+    console.log("----------------------------------------------");
+    console.log(`All pizzas have been stored to the DB`);
+    console.log("----------------------------------------------");
   }
   // handle errors in seeding
   catch(err) { 
       console.log("[ERROR] Seeding failed: ", err)
+  };
+  // --------------------------------------------------------------
+
+
+  const orderPromises = Array(2)
+  .fill(null)
+  .map(() => {
+    const orderData = {
+      order_date: faker.date.between("2021-09-11", "2021-09-13"),
+      customerID: faker.random.arrayElement( customers ),
+      items: { pizza: faker.random.arrayElement( pizzas )._id, quantity: Math.ceil(Math.random() * 9)}
```

Comment:
Wow, nicely grabbed some random quantity here. Using faker for a random number would have been to easy for ya, I guess. So nice to apply some Math.ceil hack :)